<?xml version="1.0"?>
<launch>
    <arg name="fixed_frame" default="base_link" />

    <node  pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher" output="screen">
        <param name="publish_frequency" value="200"/>
    </node>
    
    <node type="rviz" name="rviz" pkg="rviz" args="-f $(arg fixed_frame) -d $(find relaxed_ik_ros1)/rviz/relaxed_ik_viewer.rviz" />
</launch>